suite_cam6.xml

```diff
@@ -0,0 +1,361 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<suite name="cam6" version="1.0">
+  <group name="physics_bc">
+    <!--cam6:  deep=ZM, shallow=CLUBB, macrop=CLUBB, microp=MG2, radiation=RRTMG, chem=trop_mam4 -->
+    <time_split>
+      <scheme>physics_state_check</scheme> <!-- Only if state_debug_checks is True. This scheme only looks for infs/nans, and kills the model if found. -->
+      <scheme>clybry_fam_adj</scheme>      <!-- Only matters if chemistry package with Cly and Bry is used -->
+      <!-- Negative moisture/tracer check -->
+      <!-- **************************** -->
+      <scheme>qneg3</scheme>               <!-- This scheme modifies the state directly.  Combine with state check? -->
+      <!-- **************************** -->
+      <scheme>physics_state_check</scheme>      <!-- Only if state_debug_checks is True. This scheme only looks for infs/nans, and kills the model if found. -->
+      <scheme>diag_state_b4_phys_write</scheme> <!-- Diagnostics -->
+      <!-- energy and momentum fixer -->
+      <!-- **************************** -->
+      <scheme>calc_te_and_aam_budgets</scheme>
+      <scheme>check_energy_fix</scheme>                  <!--Only matters if dycore is FV (called "LR" in physics) or SE-->
+      <scheme>check_energy_cam_update_pre_chng</scheme>  <!-- Placeholder for CAM ptend and diagnostic output calculations before "chng" call. -->
+      <scheme>check_energy_chng</scheme>                 <!-- Global integral checker required for certain diagnostic outputs -->
+      <scheme>check_energy_cam_update_post_chng</scheme> <!-- Placeholder for CAM ptend and diagnostic output calculations after "chng" call. -->
+      <scheme>calc_te_and_aam_budgets</scheme>
```

Comment:
In CAM6, calc_te_and_aam_budgets is being called with different inputs for the two calls.  How will we handle this via the suite?  There should probably be a comment added to each call with the different input indicated until this is addressed.

Reply:
I am personally not sure what the best method is either for handling the different inputs, although it might be important to note that the actual calculations are done with the same inputs, it's just the diagnostic variable names in the history files that are different. 

For now, though, I agree that including the outfield variable suffixes in the scheme comments would be beneficial, so I will go ahead and do so.